x-pack/solutions/security/plugins/security_solution/public/cloud_security_posture/components/entity_insight.tsx

```diff
@@ -46,25 +54,39 @@ export const EntityInsight = <T,>({
   const { euiTheme } = useEuiTheme();
   const insightContent: React.ReactElement[] = [];
 
-  const { hasMisconfigurationFindings: showMisconfigurationsPreview } = useHasMisconfigurations(
+  const {
+    hasMisconfigurationFindings: showMisconfigurationsPreview,
+    has3PMisconfigurationFindings,
+  } = useHasMisconfigurations(field, value);
+
+  const { hasVulnerabilitiesFindings, has3PVulnerabilitiesFindings } = useHasVulnerabilities(
     field,
     value
   );
 
-  const { hasVulnerabilitiesFindings } = useHasVulnerabilities(field, value);
-
   const showVulnerabilitiesPreview = hasVulnerabilitiesFindings && field === 'host.name';
 
   const { to, from } = useGlobalTime();
 
-  const { hasNonClosedAlerts: showAlertsPreview, filteredAlertsData } = useNonClosedAlerts({
+  const {
+    hasNonClosedAlerts: showAlertsPreview,
+    filteredAlertsData,
+    alertHas3rdPartyData,
+  } = useNonClosedAlerts({
     field,
     value,
     to,
     from,
     queryId: DETECTION_RESPONSE_ALERTS_BY_STATUS_ID,
   });
 
+  const [showCallOut, setShowCallOut] = useState(!localStorage.getItem('InsightCallOutOnDismiss'));
+
+  const onDismiss = () => {
+    setShowCallOut(false);
+    localStorage.setItem('InsightCallOutOnDismiss', 'hidden');
```

Comment:
will this work? we set the value to `hidden` but the value will exist on reload, so `useState(!localStorage.getItem('InsightCallOutOnDismiss'));` wil return `true` and we will show the callout again. Am I missing smth?